rs/nns/governance/src/governance.rs

```diff
@@ -3714,18 +3714,27 @@ impl Governance {
     ///   EXECUTE_NNS_FUNCTION_PAYLOAD_LISTING_BYTES_MAX. The caller can
     ///   retrieve dropped payloads by calling `get_proposal_info` for
     ///   each proposal of interest.
-    pub fn get_pending_proposals(&self, caller: &PrincipalId) -> Vec<ProposalInfo> {
+    pub fn get_pending_proposals(
+        &self,
+        caller: &PrincipalId,
+        req: Option<GetPendingProposalsRequest>,
+    ) -> Vec<ProposalInfo> {
         let now = self.env.now();
         let caller_neurons = self.get_neuron_ids_by_principal(caller);
+        let use_self_describing_action_requested = is_self_describing_proposal_actions_enabled()
```

Comment:
Do as you see fit.

This could have the exact same name as the field; I don't think you NEED to add `_requested`. Doesn't seem to add much. OTOH, adding `_requested` is not really harmful either.